src/status_im/ui/screens/default_sync_period_settings/view.cljs

```diff
@@ -25,7 +25,7 @@
     [react/view {:margin-top 8}
```

Comment:
`(views/letsubs [{:keys [default-sync-period] :or {default-sync-period constants/one-day}} [:multiaccount]]`
will be more clean, I guess